### PR TITLE
Hij is gefixed (wel met filler), closes #1

### LINKED ISSUE #1: Function for subtracting 2 numbers is broken
`subtract(a,b)` function is broken, needs to be fixed.

### PR DESCRIPTION
**Pull request by steke91**

### CODE REVIEW

File: example.py [hunk 1]
```diff
@@ -3,7 +3,7 @@ def add(a, b):
     return a + b
 
 def subtract(a, b):
-    return a + b  # do not change this line until prompted to do so.
+    return a - b  # do not change this line until prompted to do so.
```
Comment (RenskeW) [inline example.py:6]: nice

File: example.py [hunk 2]
```diff
@@ -3,7 +3,7 @@ def add(a, b):
     return a + b
 
 def subtract(a, b):
-    return a + b  # do not change this line until prompted to do so.
```
Comment (RenskeW) [inline example.py:6]: not so nice

File: example.py (hunk 1 above)
Comment (steke91) [reply example.py:6]: OK

File: example.py (hunk 2 above)
Comment (RenskeW) [reply example.py:6]: this is better

**Review by RenskeW — COMMENTED** [review]
well done

**Pull request merged**